.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+on:
+  push:
```

Comment:
Probably no need to run on every `push`, I would say `pull_request` is sufficient. Once the test is run against a different version (on PR open) the developer will detect if there are version inconsistencies, no need to run at any successive commit push.

Reply:
:ok_man: 

I have experienced a pattern, and I'm afraid.

1. Create PR A - ok
2. Create PR B - ok
3. Merge PR A - ok
4. Merge PR B - broken, but I cannot noticed

So, I recommend enabling "Require branches to be up to date before merging" in GitHub branch protection if you are omitting the push trigger.

Reply:
That is indeed a good idea!